plugins/ltac/tacinterp.ml

```diff
@@ -1079,7 +1079,7 @@ let type_uconstr ?(flags = (constr_flags ()))
     ltac_constrs = closure.typed;
     ltac_uconstrs = closure.untyped;
     ltac_idents = closure.idents;
-    ltac_genargs = Id.Map.empty;
+    ltac_genargs = closure.genargs;
```

Comment:
https://github.com/coq/coq/blob/1a57401e2249e7c824d04afa0b3a169ee72c45fd/plugins/ltac/tacinterp.ml#L602 should be changed too?

Reply:
No, the `ist.lfun` and the value returned in the closure by `extract_ltac_constr_context` coincide. I guess we should have an API that enforces this statically but this would require a bit of refactorization.